test/fool_ai_detector/service/test_naive_baseline_processor_text.py

```diff
@@ -0,0 +1,42 @@
+import os
+from pathlib import Path
+
+from fool_ai_detector.service.naive_baseline_processor_text import NaiveBaselineProcessorText
+
+
+class TestNaiveBaselineProcessorText:
+    base_path_this_class = os.path.dirname(os.path.abspath(__file__))
+    output_generator_file_path = os.path.join(base_path_this_class, '..', '..', '..', 'src',
+                                              'fool_ai_detector', 'resources', 'ai_gen_text', 'alpen.txt')
+    output_processor_directory_path = os.path.join(base_path_this_class, '..', '..', 'test_output',
+                                                   'naive_processor_texts')
+    output_processor_file_path = os.path.join(base_path_this_class, '..', '..', 'test_output',
+                                              'naive_processor_images', 'processed_alpen.txt')
+
+    def set_up(self):
+        """
+        Helper method: This method creates the test folders.
+        """
+        if not os.path.exists(self.output_processor_directory_path):
+            os.makedirs(self.output_processor_directory_path)
+
+    def test_process(self):
+        self.set_up()
+
+        naive_baseline_processor = NaiveBaselineProcessorText()
+        naive_baseline_processor.process(self.output_generator_file_path, self.output_processor_file_path)
```

Comment:
I need to create the `processed_alpen.txt` file, else the test will fail here. Could we change it so it does create the file, if it does not exist yet?